src/test/graphql/mutations/savedItemsMutationService-delete.integration.ts

```diff
@@ -123,6 +123,7 @@ describe('Delete/Undelete SavedItem: ', () => {
 
   afterAll(async () => {
     await writeClient().destroy();
+    await readClient().destroy();
```

Comment:
This test needed the readClient destroyed to not hang them. I think its cause we made 2 queuries, 1 thats a mutation, 1 thats not. And we have logic that switches the client based on mutations. So when the query runs, it inits the readClient in the execution context, but we never cleaned it up.